File: online judge.kaner murillo/python/11764.py
#https://onlinejudge.org/external/117/11764.pdf
#11764

casos = int(input())

for i in range(1,casos+1):

    n = int(input())

    mapa = [int(x)for x in input().split()]

    if n==1:
        print(f"Case {i}: 0 0")
    
    else:
        
        saltosA = 0
        saltosB = 0

        for x in range(n-1):

            if mapa[x]>mapa[x+1]:
                
                saltosB+=1
    
            elif mapa[x]<mapa[x+1]:
                saltosA+=1
                
        print(f"Case {i}: {saltosA} {saltosB}")


    

File: online judge.kaner murillo/c++/11764.cpp
// 11764
// https://onlinejudge.org/external/117/11764.pdf

#include <iostream>
#include <cstdio>
#include<vector>

using namespace std;

int main(){
    int casos,paredes,i,j,altura,arriba,abajo;
    scanf("%i",&casos);
    vector<int>alturas;

    for(i=1;i<casos+1;i++){
        scanf("%i",&paredes);
        alturas.clear();
        arriba = 0;
        abajo = 0;

        for(j=0;j<paredes;j++){

            scanf("%i",&altura);
            alturas.push_back(altura);
        }

        for(j=0;j<paredes-1;j++){

            if(alturas[j]>alturas[j+1]){
                
                arriba+=1;
            }

            else if(alturas[j]<alturas[j+1]){
                abajo+=1;
            }
        }
        
        printf("Case %i: %i %i\n",i ,abajo ,arriba);
    }
    return 0;
}